core/src/main/scala/org/apache/spark/rdd/JdbcRDD.scala

```diff
@@ -70,7 +71,8 @@ class JdbcRDD[T: ClassTag](
     }).toArray
   }
 
-  override def compute(thePart: Partition, context: TaskContext) = new NextIterator[T] {
+  override def compute(thePart: Partition, context: TaskContext): Iterator[T] = new NextIterator[T]
+  {
```

Comment:
This brace on a line by itself looks a bit funny to me; maybe we could split at the argument list instead?